src/dds/rsdds-serializable.cpp

```diff
@@ -0,0 +1,171 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#include "rsdds-serializable.h"
+
+#include <src/serialized-utilities.h>
+#include <src/core/sensor-interface.h>
+#include <src/core/device-interface.h>
+
+#include <rsutils/json.h>
+#include <set>
+
+using json = nlohmann::json;
+
+
+namespace librealsense {
+
+
+dds_serializable::dds_serializable()
```

Comment:
It is better to remove empty constructors and let the compiler create default

Reply:
Better how? But OK, removed it.